source/timer_sys.h

```diff
@@ -16,10 +16,6 @@
 #ifndef _PL_NANO_TIMER_SYS_H_
 #define _PL_NANO_TIMER_SYS_H_
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
 /**
```

Comment:
These got removed accidentally, not sure if these are needed now so I will add these back.